how-to-guides/hello-neptune/scripts/hello_neptune.py

```diff
@@ -5,6 +5,7 @@
     project="common/quickstarts",
     api_token=neptune.ANONYMOUS_API_TOKEN,
     tags=["quickstart", "script"],
+    dependencies="infer",  # to infer dependencies. You can also pass the path to the requirements.txt file
```

Comment:
Given that this `"infer"` option seems a bit error-prone, I don't think we should use it here. I also had problems with pipreqs in my environment and I'd like for the quickstart to be as waterproof as possible.

Reply:
@normandy7 can you describe the issue you faced? Worst case scenario: it might not get the right dependencies or miss out on some (for some edge cases). Do you think this might becomes a blocker or halt the run? This can be a good way of gathering feedback and iterating on the current implementation, if `pipreqs` is that error-prone or problematic. Again I leave if up to you to decide if this might be a major regression when it comes to the first introduction to the product.

Reply:
In my case it errored and didn't capture anything. (I've shared the details with Siddhant.)

It's a brand new feature, so I strongly feel we should let it mature a bit before putting it in the quickstart.

Reply:
@parthpankajtiwary It's this error that can cause the script to crash: https://stackoverflow.com/questions/51503913/pipreqs-unicodedecodeerror-charmap-codec-cant-decode-byte-0x98-in-position
It does not always happen (I could run the neptune script fine), but I've faced this issue using pipreqs independently.
Adding `encoding=utf8` in the subprocess call should take care of this.

Reply:
@normandy7 @SiddhantSadangi sure, let's keep it out of the quickstart for now. @SiddhantSadangi possible to batch these instances and send it my way? Inability to reproduce these instances is definitely a major drawback atm.

Reply:
I could not reproduce this specific error using the neptune client on my system, and tests ran fine. So something to do with Sabine's settings imo.
However, since this can cause the script to crash when it does occur, definitely worth excluding from the quickstart until it is more stable.

Parking this PR for now.